mlflow/store/tracking/rest_store.py

```diff
@@ -55,8 +56,13 @@ def __init__(self, get_host_creds):
         super().__init__()
         self.get_host_creds = get_host_creds
 
-    def _call_endpoint(self, api, json_body):
-        endpoint, method = _METHOD_TO_INFO[api]
+    def _call_endpoint(self, api, json_body, endpoint=None):
+        if endpoint:
```

Comment:
This is the only non-obvious change in the PR - earlier we don't have dynamic endpoints, but the `GetTraceInfo` requires setting endpoint based on `trace_id`.